packages/e2e/cypress/e2e/api/dashboard.cy.ts

```diff
@@ -163,24 +164,22 @@ describe('Lightdash dashboard', () => {
                     },
                 ).then(({ chart: newChart2, dashboard: updatedDashboard2 }) => {
                     expect(updatedDashboard2.tiles.length).to.eq(2);
-                    const firstTile = updatedDashboard2
-                        .tiles[0] as DashboardChartTile;
-                    const secondTile = updatedDashboard2
-                        .tiles[1] as DashboardChartTile;
-
-                    expect(
-                        firstTile.properties.savedChartUuid,
-                        "Check if first tile didn't change",
-                    ).to.eq(newChart.uuid);
-                    // assert second tile is correct
-                    expect(
-                        secondTile.properties.savedChartUuid,
-                        'Check if second tile is correct',
-                    ).to.eq(newChart2.uuid);
-                    expect(
-                        secondTile.properties.belongsToDashboard,
-                        'Check if second tile belongs to a dashboard',
-                    ).to.eq(true);
+                    const firstTile = updatedDashboard2.tiles.find(
```

Comment:
so the orders of the tiles are not consistent ?

Reply:
correct, we insert the tiles in parallel into the database so one can be faster than the other. The order in the array is not relevant for the UI as the library figures the position based on the x/y h/w properties.